module-system/module-implementations/sov-ibc/src/lib.rs

```diff
@@ -10,31 +10,89 @@ mod router;
 
 use codec::ProtobufCodec;
 use context::clients::{AnyClientState, AnyConsensusState};
-use ibc::core::ics24_host::identifier::ClientId;
-use ibc::core::ics24_host::path::ClientConsensusStatePath;
+use ibc::core::events::IbcEvent;
+use ibc::core::ics03_connection::connection::ConnectionEnd;
+use ibc::core::ics04_channel::channel::ChannelEnd;
+use ibc::core::ics04_channel::commitment::{AcknowledgementCommitment, PacketCommitment};
+use ibc::core::ics04_channel::packet::{Receipt, Sequence};
+use ibc::core::ics24_host::identifier::{ClientId, ConnectionId};
+use ibc::core::ics24_host::path::{
+    AckPath, ChannelEndPath, ClientConnectionPath, ClientConsensusStatePath, CommitmentPath,
+    ConnectionPath, ReceiptPath, SeqAckPath, SeqRecvPath, SeqSendPath,
+};
+use ibc::core::timestamp::Timestamp;
+use ibc::Height;
 use sov_modules_api::Error;
 use sov_modules_macros::ModuleInfo;
 use sov_state::WorkingSet;
 
 pub struct ExampleModuleConfig {}
 
 #[derive(ModuleInfo)]
-pub struct IbcModule<C: sov_modules_api::Context> {
+pub struct Ibc<C: sov_modules_api::Context> {
```

Comment:
Can you add a comment about why this *needs* to be `Ibc` (in relation to commitment prefix)?

Reply:
[6a8bebe](https://github.com/informalsystems/sovereign-sdk/pull/1/commits/6a8bebe6a9559c0e2c69d9535fb63b132dab9d47)